docs/resources/pipeline_schedule_variable.md

```diff
@@ -24,8 +24,8 @@ resource "gitlab_pipeline_schedule" "example" {
 }
 
 resource "gitlab_pipeline_schedule_variable" "example" {
```

Comment:
This documentation example originates at:

https://github.com/gitlabhq/terraform-provider-gitlab/blob/ee81e298d6d33ebbbaa762e2624920fbf6022f22/examples/resources/gitlab_pipeline_schedule_variable/resource.tf#L9

Do you mind changing it there and run `make reviewable` or `make generate` to re-generate this doc file here?

The PR looks good otherwise 🎉